airflow/sensors/base.py

```diff
@@ -330,6 +330,16 @@ def reschedule(self):
     def get_serialized_fields(cls):
         return super().get_serialized_fields() | {"reschedule"}
 
+    def raise_failed_or_skiping_exception(self, failed_message: str, skipping_message: str = "") -> None:
+        """Raise AirflowSkipException if self.soft_fail is set to True. Otherwise raise AirflowException."""
+        if self.soft_fail:
+            if skipping_message:
+                message = skipping_message
+            else:
+                message = failed_message
+            raise AirflowSkipException(message)
```

Comment:
```suggestion
            raise AirflowSkipException(skipping_message or failed_message)
```

Reply:
But, since `skipping_message` is always `{failed_message}{self._skipping_message_postfix}`, why not just do it in this function?

Reply:
It's not always `{failed_message}{self._skipping_message_postfix}`. e.g., https://github.com/apache/airflow/pull/33196/files#diff-257fa1956f6fafd5d9136377b3d84c795882835712892903ff610e37ed688922R289-R293

Even though I think doing it in this function would be ideal, it changes the message of existing messages. (I guess this is likely the case for other sensors I've not yet started to work on as well)

Reply:
Just updated the code as suggestion